### ISSUE TITLE
Word wrap only works on the right side of the application.

### ISSUE THREAD

**Issue by jbleuzen** [ProxymanApp/Text-Differ#6]
I noticed that the word-wrap option only works for the right side of the application.
When I've disabled then re-enabled the setting but it stays the same, the left side is not wrapped…

![CleanShot 2024-02-05 at 14 41 16](https://github.com/ProxymanApp/Text-Differ/assets/280681/ea3a5e03-54b5-4f97-8962-8be7acedc00f)

Maybe I'm wrong but I would expect that both sides are using the same setting, so left side should be word-wrapped.

**Comment by NghiaTranUIT** [comment]
Sorry for the late reply @jbleuzen.

It's how Monaco Editor works when using Word Wrap + Side By Side Mode.

If you switch to Inline mode, it's easier to see

![Screenshot 2024-03-12 at 10 42 42](https://github.com/ProxymanApp/Text-Differ/assets/5878421/69fbd6ac-602c-4430-97bc-f6944a5d3172)

**Comment by DunhamGitHub** [comment]
Yeah. The app is not usable in this state. _(Got the Appstore version just now, will refund)._

How it should look (how to actually spot changes):
<img width="823" alt="Image" src="https://github.com/user-attachments/assets/f61841b4-7ce1-4c45-b9b2-c31c5095ecd9" />

How it does look (how changes are hard to see):
<img width="1293" alt="Image" src="https://github.com/user-attachments/assets/1d9f5b7b-1d4e-4212-84ae-a44256c1a978" />
